js/src/forum/components/VotesUserPage.ts

```diff
@@ -0,0 +1,25 @@
+import app from 'flarum/forum/app';
+import PostsUserPage from 'flarum/forum/components/PostsUserPage';
+
+/**
+ * The `VotesUserPage` component shows posts which user voted on.
+ */
+export default class VotesUserPage extends PostsUserPage {
+  /**
+   * Load a new page of the user's activity feed.
+   *
+   * @param {Integer} [offset] The position to start getting results from.
+   * @return {Promise}
```

Comment:
If we've done TS, we can just scrap this :)

```suggestion
   * @param offset The position to start getting results from.
```